SI/Sem.07/Pract.07/References/Task06.cpp

```diff
@@ -0,0 +1,83 @@
+﻿#include <iostream>
```

Comment:
The solution doesn't use references? Try to solve it without using arrays and use references